src/utils/index.tsx

```diff
@@ -12,6 +12,21 @@ const addToStorage = (item: string, data: any) => {
   localStorage.setItem(item, JSON.stringify(data))
 }
 
+export function findCartItem(name: string) {
+  const storageEvent = new Event('storage')
+  const previusData: Array<CartItem> = getStorageItem('cart')
+
+  const currentCartValue = previusData ? previusData : []
```

Comment:
Aopaaa! 🤠 
Capitão, utilizando `||` ou o `??` não rolaria?

```suggestion
  const currentCartValue = previusData || []
```

Reply:
lindo